lib/environment/history.js

```diff
@@ -11,3 +11,11 @@ export async function addURLToHistory(url) {
 
 	await sendMessage('addURLToHistory', url);
 }
+
+/**
+ * @param {...string} url
+ * @returns {Promise<string[], *>}
+ */
+export async function getVisitedURLs(...urls) {
+	return await sendMessage('getVisitedURLs', urls);
```

Comment:
`async`/`await` here are redundant, we can just return the promise returned by sendMessage directly.